GoogleForCreators/web-stories-wp
# Generalize Pill component

## Summary
Creates `PillGroup` and `Pill` components for general use instead of specifically for Media only.


## Relevant Technical Choices



## To-do

- [x] Verify matching with design



## User-facing changes
N/A (Behind the feature flag)


## Testing Instructions
- Open 3rd party Media tab
- Verify that the Category Pills filter the Media as expected when clicking on it.
- Verify that it's possible to navigate in the categories using keyboard (up-down-left-right, selecting with Space/Enter)
- Verify that the selected Category is visibly different from the unselected ones.


---



Fixes #3967
